benchmarks/prototype/moe_training/benchmark_scaled_grouped_mm_dq.py

```diff
@@ -28,6 +28,9 @@
 # Needed since changing args to function causes recompiles
 torch._dynamo.config.cache_size_limit = 1000
 
+# Workaround for https://github.com/pytorch/ao/pull/2990#issuecomment-3285762681
+torch._dynamo.config.automatic_dynamic_shapes = False
```

Comment:
is this needed for fast e2e? if yes, add to README.md?

Reply:
No we dont need to recommend in the readme. Benchmarks artificially hit automatic dynamic shapes because we are sweeping certain dims. In practice the dims that are being swept won't be dynamic or if they are then its up to the users to decide how they want to handle this -> dynamic=false w/ more recompiles or the default behavior